### DISCUSSION TITLE
What is the Slack Community URL?

### DISCUSSION THREAD

**Discussion by jwlai** [Project-Books/book-project#695] [Q&A] [answered]
Hi, my name is Justin and I took a look at your project! I work as a Software Engineer during the day but am interested in picking up some Open Source Software Experience. The tech stack that you mention is up my alley - Microservice, Java, React, GraphQL, Docker. I've taken a look at the documentation and would like to join the Slack to read more before diving in! 

Thanks! Love the project idea- looking forward to seeing this app deployed sometime!

**Comment by knjk04** [comment, marked as answer]
Hi Justin, welcome to the Book Project & thanks for the kind words! 

Here's the Slack link: https://join.slack.com/t/teambookproject/shared_invite/zt-punc8os7-Iz9PTCAkYcO_0S~XwtO5_A

**Reply by knjk04** [reply]
@jwlai If you're interested after you've completed your currently assigned issue, there's an available Docker issue: #632
